docs/configuration/schema.yaml

```diff
@@ -191,13 +199,23 @@ properties:
               properties:
                 matches:
                   type: string
-                  description: The URL pattern to match. For example, `https://api.example.com/v2/**` or `https://example.com/*/test.yaml`
+                  description: The URL pattern to match.
+                  example: https://api.example.com/**
                 name:
                   type: string
-                  description: The header name. For example, `X-API-KEY`.
+                  description: The HTTP header name.
+                  example: X-API-KEY
                 value:
                   type: string
-                  description: The value of the header. We recommend to use `envVariable` instead for any secrets.
+                  description: The value of the header. Mutually exclusive with `envVariable`.
+                  example: 123-abc
                 envVariable:
                   type: string
-                  description: The environment variable name which will be resolved and should contain the value.
+                  description: The name of the environment variable that contains the value of the header. Mutually exclusive with `value`.
+                  example: SECRET_KEY
+  features.openapi:
```

Comment:
What is the reason this is moved to the bottom?

Reply:
I wanted it to be the same order as the `redocly.yaml` sections are presented in `configuration-file.mdx` which is:

`organization`
`apis`
`lint`
`features.mockServer`
`region`
`resolve`
`features.openapi`